### ISSUE TITLE
Bug: UploadMedia not work - NoSuchMethodError exception

### ISSUE THREAD

**Issue by pcomp96** [Bindambc/whatsapp-business-java-api#92]
Hi @Bindambc ,

I implemented everything easily and now I'm doing the testing phase, a week ago I launched the tests for the uploadMedia and it worked. Today launching a test with the average upload, it failed.
The exception thrown comes from the library

java.lang.NoSuchMethodError: okhttp3.MultipartBody.create([BLokhttp3/MediaType;)Lokhttp3/RequestBody; at com.whatsapp.api.impl.WhatsappBusinessCloudApi.uploadMedia([WhatsappBusinessCloudApi.java:73]

[Go to WhatsappBusinessCloudApi.uploadMedia](https://github.com/Bindambc/whatsapp-business-java-api/blob/6e4f525402c5ddaf271ae9eeec49ce9f64bbde51/src/main/java/com/whatsapp/api/impl/WhatsappBusinessCloudApi.java#L69)

```java
  public UploadResponse uploadMedia(String phoneNumberId, String fileName, FileType fileType, byte[] file) {

        RequestBody requestFile = MultipartBody.create(file, MediaType.parse(fileType.getType()));

           .......
    }
```

From the source code I see that there is no method with this signature:
```java
MultipartBody.create(byte[] file, MediaType fileType);
```

There is a method that has this signature and I think you need to change this line so as not to throw this exception.

![code](https://user-images.githubusercontent.com/80928861/236167135-95ea3064-0d6f-4353-ac2b-c2d5881cabfe.png)

For the solution I think it would be enough just to reverse the parameters passed to the method.

I hope you find this observation useful, I remain available for clarification .. Thank you! 😉

**Comment by Bindambc** [comment]
Hello @pcomp96 , I couldn't reproduce the exception you mentioned. I am using this example: ["UploadMediaExample.java"](https://github.com/Bindambc/whatsapp-business-java-api/blob/main/src/test/java/com/whatsapp/api/examples/UploadMediaExample.java), and it worked fine. Could you please send the code snippet where you execute the upload method?
![image](https://user-images.githubusercontent.com/7831956/236312374-7e63e228-41c0-4dc1-81ef-b4987b553ce0.png)

**Comment by pcomp96** [comment]
Hi @Bindambc ,
Pardon for the late reply, but I am working on other things at the moment and have delegated my developments to some colleagues. 

I launched the application and my tests and I realized that previously an application had been launched with a dependency to okhttp3 version `3.14.9` and when I compiled the application I still do not understand why it used that version and not the one used in your pom. I think it's a matter of IDE caching.

Sorry again for this useless issue that I opened.

Thanks again!
